src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/DownloadModal/components/DownloadMenuSelection/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { Tooltip } from "czifui";
+import { ReactNode, useState } from "react";
+import {
+  CheckBoxInfo,
+  CheckboxLabel,
+  DownloadType,
+  DownloadTypeInfo,
+  StyledCheckbox,
+  StyledFileTypeItem,
+} from "./style";
+
+interface Props {
+  id: string;
+  isChecked: boolean;
+  isDisabled?: boolean;
+  onChange(): void;
+  downloadTitle: string;
+  fileTypes: string;
+  children: ReactNode;
+  tooltipTitle?: ReactNode;
+}
+
+const DownloadMenuSelection = ({
+  id,
+  isChecked,
+  isDisabled,
+  onChange,
+  downloadTitle,
+  fileTypes,
+  children,
+  tooltipTitle,
+}: Props): JSX.Element => {
+  const [anchorEl, setAnchorEl] = useState<HTMLElement>();
+
+  const menuItem = (
+    <StyledFileTypeItem isSelected={isChecked} isDisabled={isDisabled}>
+      <CheckBoxInfo>
+        <StyledCheckbox
+          id={id}
+          disabled={isDisabled}
+          onChange={onChange}
+          stage={isChecked ? "checked" : "unchecked"}
+        />
+      </CheckBoxInfo>
+      <CheckboxLabel htmlFor={id}>
+        <span onMouseOver={(e) => setAnchorEl(e.currentTarget)}>
+          <DownloadType>{downloadTitle} </DownloadType> ({fileTypes})
```

Comment:
Nit
```suggestion
          <DownloadType>{downloadTitle}</DownloadType> ({fileTypes})
```

Reply:
the space is necessary for separating the bold text from the regular weight text